misc/dbt-materialize/dbt/include/materialize/macros/deploy/deploy_init.sql

```diff
@@ -44,12 +44,13 @@
 {% endfor %}
 
 {% for cluster in clusters %}
-    {% if not cluster_exists(cluster) %}
-        {{ exceptions.raise_compiler_error("Production cluster " ~ cluster ~ " does not exist") }}
+    {% set source_cluster = adapter.generate_final_cluster_name(cluster, force_deploy_suffix=False) %}
```

Comment:
`source_cluster` is maybe too load bearing? We could use something more generic like `deploy_cluster`, which also ties in better with the final outcome of suffixing the selected cluster with `dbt_deploy`.

Reply:
Should've checked the whole code, sorry. Maybe `origin_cluster` or `config_cluster`, then?

Reply:
Nea! Sounds good! Will update this now!